src/functionaltests/java/com/ericsson/ei/subscriptions/bulk/SubscriptionBulkSteps.java

```diff
@@ -23,7 +23,9 @@
 import cucumber.api.java.en.When;
 
 @Ignore
-@TestPropertySource(properties = {"logging.level.com.ericsson.ei.subscriptions.bulk=DEBUG"})
+@TestPropertySource(properties = { "logging.level.com.ericsson.ei.subscriptions.bulk=DEBUG",
```

Comment:
is the added logging level needed?

Reply:
Yes, Debug should not be DEBUG enabled,,, will probably cause too huge logs in Travis-CI. You can maybe set it to OFF as default, and if user want to rerun the test with debug enabled, user can change this parameter from OFF to DEBUG.

Reply:
Done